### ISSUE TITLE
unused_types, false positive

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#2355]
`incl` is used, but reported as unused,

```abap
TYPES: BEGIN OF incl,
  fieldname(30),
  END OF incl.

DATA BEGIN OF x031l.
INCLUDE TYPE incl.
DATA END OF x031l.
```
